vxfreeswitch/voice.py

```diff
@@ -150,10 +154,15 @@ def _success(ev):
             response = Deferred()
             self.job_queue[ev.Job_UUID] = response
             return response
+
+        def _error(err):
+            raise ClientConnectError(err)
```

Comment:
``err`` will be a ``Failure`` instance so passing it to ``ClientConnectError`` is probably not that sensible.